src/_data/defaultSchedule.yml

```diff
@@ -121,21 +117,21 @@ friday:
     name: Project documentation
     url: /course/handbook/project-documentation
   - start: 14:30
-    end: 15:15
+    end: 15:00
+    name: Presentation prep
+    url: /course/handbook/projects/#project-presentation
+  - start: 15:00
+    end: 15:30
+    name: Team SGC
+    url: /course/handbook/retrospectives/#team-retrospectives
+  - start: 15:30
+    end: 16:15
```

Comment:
Maybe having SGC before presentations makes sense so they can talk about their workflow in their presentation?